.circleci/config.yml

```diff
@@ -144,23 +166,53 @@ workflows:
   build:
     jobs:
       - gradle/test:
-          <<: *matrix_executors
-          <<: *filters_always
+          <<: *filters_not_forked
+          matrix:
+            alias: test
+            <<: *matrix_executors
+          <<: *slack_notify_failed_main
+          context: Slack
       - build:
-          <<: *filters_always
+          <<: *filters_not_forked
           requires:
-            - gradle/test
+            - test
+          <<: *slack_notify_failed_main
+          context: Slack
       - smoke_test:
-          <<: *filters_always
+          <<: *filters_not_forked
           requires:
             - build
+          <<: *slack_notify_failed_main
+          context: Slack
       - publish_maven:
           <<: *filters_main_only
```

Comment:
I think this should be `filters_tags_only` too

Reply:
we changed that to publish on all main builds so we get the SNAPSHOTs in Maven and verify our release process continuously